sorc/chgres_cube.fd/wam_climo_data.f90

```diff
@@ -0,0 +1,2758 @@
+!! @param [in] glong - geodetic longitude(deg)                                
+!! @param [in] stl - local apparent solar time(hrs)                           
+!! @param [in] f107a - 3 month average of f10.7 flux                          
+!! @param [in] f107 - daily f10.7 flux for previous day                       
+!! @param [in] ap - magnetic index(daily) or when sw(9)=-1. :                 
+!!                - array containing:                                         
+!!                  (1) daily ap                                              
+!!                  (2) 3 hr ap index for current time                        
+!!                  (3) 3 hr ap index for 3 hrs before current time           
+!!                  (4) 3 hr ap index for 6 hrs before current time           
+!!                  (5) 3 hr ap index for 9 hrs before current time           
+!!                  (6) average of eight 3 hr ap indicies from 12 to 33 hrs pr
+!!                         to current time                                    
+!!                  (7) average of eight 3 hr ap indicies from 36 to 59 hrs pr
+!!                         to current time                                    
+!! @param [in] mass - mass number (only density for selected gas is
+!!                calculated.  mass 0 is temperature.  mass 48 for all.
+!!                mass 17 is anomalous o only.)
+!!                                                                      
+!!    Notes on input variables:                                         
+!!       ut, local time, and longitude are used independently in the    
+!!       model and are not of equal importance for every situation.     
+!!       for the most physically realistic calculation these three      
+!!       variables should be consistent (stl=sec/3600+glong/15).        
+!!       the equation of time departures from the above formula         
+!!       for apparent local time can be included if available but       
+!!       are of minor importance.                                       
+!!                                                                      
+!!       f107 and f107a values used to generate the model correspond    
+!!       to the 10.7 cm radio flux at the actual distance of the earth  
+!!       from the sun rather than the radio flux at 1 au. the following 
+!!       site provides both classes of values:                          
+!!       ftp://ftp.ngdc.noaa.gov/stp/solar_data/solar_radio/flux/       
+!!                                                                      
+!!       f107, f107a, and ap effects are neither large nor well         
+!!       established below 80 km and these parameters should be set to  
+!!       150., 150., and 4. respectively.                               
+!!                                                                      
+!! output variables:                                                 
+!! @param [out] d density array with length of 8
```

Comment:
Please list all the params together, in order they appear in the parameter list, and let doxygen highlight which ones are input, output, or inout.

Reply:
I have no pushed yet, I am using -DENABLE_DOCS=ON in Hera to check and fix. I didn't get the similar doxygen error/warning as shown from CI(?) here. Thus it comes to your good question: "what version of Doxygen?" I am using... In HERA, DOXYGEN didn;t show under "module list" and I cannot find it under "module spider".  How can I load it to check/fix code in HERA?

Reply:
The cmake build will find doxygen and print out the version number.

Reply:
The version of doxygen which is canonical is the one distributed by ubuntu-latest. That's what we use for our CI, which does the checking.